custom_connectors/custom_auth/trackvia_connector.rb

```diff
@@ -0,0 +1,712 @@
+# frozen_string_literal: true
```

Comment:
Remove `# frozen_string_literal: true` and empty 2nd line.
We ignore this and `Block has too many lines` and += operations..